Pocket/pocket-ios
# feat(user): updating user query

## Summary
* move isPremium to own query

## References 
* [IN-1198](https://getpocket.atlassian.net/browse/IN-1198)

## Implementation Details
* moved isPremium out of fetch saves and into its own query

## Test Steps
* as non-premium user, I should see all upsells
* as premium user, I should not see any upsells

## PR Checklist:

## Screenshots


[IN-1198]: https://getpocket.atlassian.net/browse/IN-1198?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
